### PR TITLE
Smithing fix

### PR DESCRIPTION
**Pull request by Pokebutt**
## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
added an extra condition for smithed weapons so that they function with eris code
/:cl:

### CODE REVIEW

File: code/modules/smithing/finished_items.dm [hunk 1]
```diff
@@ -325,7 +324,7 @@
 	icon_prefix = "katana_smith"
 	overlay_state = "hilt_katana"
 	force = WEAPON_FORCE_SLASH_LARGE
-	wielded_mult = WEAPON_SLASH_TWOHAND_MULT
+	wielded_mult = 1.5
```
Comment (Superlagg) [inline code/modules/smithing/finished_items.dm:327]: Please make this a define so its easier to change later. You can change the original defined value if you want.

File: code/modules/smithing/finished_items.dm [hunk 2]
```diff
@@ -356,7 +355,7 @@
 	overlay_state = "shaft_axe"
 	total_mass = TOTAL_MASS_MEDIEVAL_WEAPON * 2
 	force = WEAPON_FORCE_AXE_LARGE
-	wielded_mult = WEAPON_AXE_TWOHAND_MULT
+	wielded_mult = 1.8
```
Comment (Superlagg) [inline code/modules/smithing/finished_items.dm:358]: This too

File: modular_coyote/eris/code/wielding.dm [hunk 3]
```diff
@@ -21,6 +21,8 @@
 	wielded = FALSE
 	if(force_unwielded)
 		force = force_unwielded
+	if(wielded_mult)
+		force = (force / wielded_mult)
 	else
 		force = (force / 1.15)
```
Comment (Superlagg) [inline modular_coyote/eris/code/wielding.dm:27]: I know its not part of your PR, but while you're at it, would you please turn this 1.15 into a define? Could go in the file with the other weapon damage defines.

File: modular_coyote/eris/code/wielding.dm [hunk 4]
```diff
@@ -7,7 +7,7 @@
 	var/wielded_icon //The item state used when it's wielded. Guns are snowflakey and have their own shit for this. This is for non guns.
 	var/force_unwielded = 0 //If you have a specific force for it being unwielded. If for whatever reason you don't want to use the original force of the weapon.
 	var/force_wielded = 0 //If you have a specific force for it being wielded.
-
+	var/wielded_mult = 1
```
Comment (Superlagg) [inline modular_coyote/eris/code/wielding.dm:10]: Would having this always set to 1 by default mean that non-smithed weapons would get no benefit from being wielded?

**Review by Superlagg — CHANGES_REQUESTED** [review]

File: modular_coyote/eris/code/wielding.dm (hunk 4 above)
Comment (Pokebutt) [reply modular_coyote/eris/code/wielding.dm:10]: that was the default legit

Comment (Pokebutt) [reply modular_coyote/eris/code/wielding.dm:10]: also I believe not, it did apply normally

**Review by Superlagg — APPROVED** [review]
Looks good!

**Pull request merged**